src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunnerPool.cs

```diff
@@ -50,10 +48,7 @@ public TestRunResult TestMultipleMutants(ITimeoutValueCalculator timeoutCalc, IR
             }
         }
 
-        public TestRunResult RunAll(ITimeoutValueCalculator timeoutMs, Mutant activeMutant, TestUpdateHandler update)
-        {
-            return TestMultipleMutants(timeoutMs, activeMutant == null ? null : new List<Mutant> { activeMutant }, update);
-        }
+        public TestRunResult RunAll(ITimeoutValueCalculator timeoutMs, Mutant activeMutant, TestUpdateHandler update) => TestMultipleMutants(timeoutMs, activeMutant == null ? null : new List<Mutant> { activeMutant }, update);
```

Comment:
Wrapping this in one line does not really make it more readable

Reply:
Thanks for pointing to this. It turns out this method should have been removed altogether; it was a remnant of the early design which did not support testing multiple mutants and was needed while Stryker still supported `dotnet test`  on top of VsTest